proteus/mprans/SpatialTools.py

```diff
@@ -1390,7 +1385,7 @@ def assembleAuxiliaryVariables(domain):
     -----
     Should be called after assembleGeometry
     """
-    domain.auxiliaryVariables = []
+    domain.auxiliaryVariables = {'twp': [], 'vof': []}
```

Comment:
I put out a question on slack to this aim, but I'll ask it here too for redundancy: can we be sure that we know all the keys that we might need?

I could put in `redist` and `ls` in the `assembleAuxiliaryVariables(domain)` dictionary initialization.  To be cautious, I could add in `dissipation`, `ls_consrv`, and `kappa`.  If any new models get tacked on, though, I'll have to return to this again (and if I'm a user who isn't closely connected to Proteus, I'm going to have to maintain a local branch with my changes each time).

If the above keys are pretty much final, then I'm fine with that.  If not, I'd rather use `domain.auxiliaryVariables.get(*key name*, [ ])` and not initialize anything unnecessarily in the dictionary (but have methods dynamically add keys for it to initialize, potentially into a list).